### ISSUE TITLE
No matching distribution found for mysql-connector==2.1.4

### ISSUE THREAD

**Issue by BarbzYHOOL** [ShahriyarR/MySQL-AutoXtraBackup#366]
I try to install an old version (because I have configured everything for it, I will upgrade later) but it says this:

```
pip3 install mysql-autoxtrabackup==1.5.2
Collecting mysql-autoxtrabackup==1.5.2
  Downloading https://files.pythonhosted.org/packages/05/ef/b2c483787455e397ec64dfc382d0cf1e18942ce50a0d75ad5e7fdc829c30/mysql_autoxtrabackup-1.5.2-py3-none-any.whl (51kB)
    100% |████████████████████████████████| 61kB 1.8MB/s 
Collecting click>=3.3 (from mysql-autoxtrabackup==1.5.2)
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
    100% |████████████████████████████████| 81kB 2.4MB/s 
Collecting humanfriendly>=2.0 (from mysql-autoxtrabackup==1.5.2)
  Downloading https://files.pythonhosted.org/packages/90/df/88bff450f333114680698dc4aac7506ff7cab164b794461906de31998665/humanfriendly-4.18-py2.py3-none-any.whl (73kB)
    100% |████████████████████████████████| 81kB 3.3MB/s 
Collecting pid>=2.0 (from mysql-autoxtrabackup==1.5.2)
  Downloading https://files.pythonhosted.org/packages/82/bc/3633e94577c0f64864684be5a73251f194fd8673fb7c1f095597ef34dbc2/pid-2.2.5-py2.py3-none-any.whl
Collecting pytest (from mysql-autoxtrabackup==1.5.2)
  Downloading https://files.pythonhosted.org/packages/ca/e1/2f229554e5c273962fae8b286395d5bbcc7bef276d2b40e1bad954993db2/pytest-5.1.3-py3-none-any.whl (224kB)
    100% |████████████████████████████████| 225kB 1.9MB/s 
Collecting mysql-connector==2.1.4 (from mysql-autoxtrabackup==1.5.2)
  Could not find a version that satisfies the requirement mysql-connector==2.1.4 (from mysql-autoxtrabackup==1.5.2) (from versions: 2.1.7, 2.2.9)
No matching distribution found for mysql-connector==2.1.4 (from mysql-autoxtrabackup==1.5.2)

```

Can you tell me how to fix this?

**Comment by ShahriyarR** [comment]
Please pull the changes and rerun.

**Comment by BarbzYHOOL** [comment]
how am I supposed to test since I was installing it from pip??

I can try from source but that's not the same, is it?

**Comment by BarbzYHOOL** [comment]
```
Installed /usr/local/lib/python3.5/dist-packages/more_itertools-7.2.0-py3.5.egg
Searching for importlib-metadata>=0.12; python_version < "3.8"
Reading https://pypi.python.org/simple/importlib-metadata/
Downloading https://files.pythonhosted.org/packages/5d/44/636bcd15697791943e2dedda0dbe098d8530a38d113b202817133e0b06c0/importlib_metadata-0.23.tar.gz#sha256=aa18d7378b00b40847790e7c27e11673d7fed219354109d0e7b9e5b25dc3ad26
Best match: importlib-metadata 0.23
Processing importlib_metadata-0.23.tar.gz
Writing /tmp/easy_install-tjcekw0d/importlib_metadata-0.23/setup.cfg
Running importlib_metadata-0.23/setup.py -q bdist_egg --dist-dir /tmp/easy_install-tjcekw0d/importlib_metadata-0.23/egg-dist-tmp-t4576w19
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'use_scm_version'
  warnings.warn(msg)
warning: no previously-included files found matching '.gitignore'
no previously-included directories found matching 'build'
no previously-included directories found matching '.tox'
zip_safe flag not set; analyzing archive contents...
Moving importlib_metadata-0.0.0-py3.5.egg to /usr/local/lib/python3.5/dist-packages
Adding importlib-metadata 0.0.0 to easy-install.pth file

Installed /usr/local/lib/python3.5/dist-packages/importlib_metadata-0.0.0-py3.5.egg
error: The 'importlib-metadata>=0.12; python_version < "3.8"' distribution was not found and is required by pytest, pluggy
--------------------------------------------------------------------------------------------------------------------------------
```

@ShahriyarR two errors above, also I installed with "python3 setup.py" which is not my usual way of doing it

**Comment by BarbzYHOOL** [comment]
alright, I tried to install with pip instead

`pip3 install git+https://github.com/ShahriyarR/MySQL-AutoXtraBackup@release_v1.5.4` effectively had the same bug

`pip3 install git+https://github.com/ShahriyarR/MySQL-AutoXtraBackup` (= master branch) worked fine ! so that critical bug is fixed, you can release 1.5.5 imo

**Comment by BarbzYHOOL** [comment]
@ShahriyarR or 1.5.4.b :)
